### ISSUE TITLE
Dependencies won't install if description contains new lines

### ISSUE THREAD

**Issue by sirex** [python-poetry/poetry#3610]
- [x] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**:  Manjaro Linux 20.2.1
- **Poetry version**: 1.1.4

## Issue

When I create `pyproject.toml` file, that looks like this:

```toml
[tool.poetry]
name = "poetry-bug-repro"
version = "0.1.0"
authors = ["Me <me@example.com>"]
description = """\
Poetry bug repro.
"""
[tool.poetry.dependencies]
python = "^3.9"
requests = "*"
```

(Note the new line in the description). Then, when I `poetry build` it, and try to install wheel package, it won't install any dependencies, in this case it won't install `requests`.

But if I replace description to:

```toml
description = "Poetry bug repro."
```

Then dependencies are installed as expected.

I think, the issue is that when `.dist-info/METADATA` file is generated, it creates an empty line like this:

```yaml
Metadata-Version: 2.1
Name: poetry-bug-repro
Version: 0.1.0
Summary: Poetry bug repro.

Author: Me
Author-email: me@example.com
Requires-Python: >=3.9,<4.0
Requires-Dist: requests
```

After this new line, after `Summary`, everything else is treated as package description, I guess.

**Comment by sinoroc** [comment]
This should be only one line:
* https://packaging.python.org/specifications/core-metadata/#summary

**Comment by sirex** [comment]
That is fine, if it should be only one line. But Poetry should refuse to build a corrupted package if given summary is not one line. At least, it should strip summary new line is only at the end of string.

**Comment by sinoroc** [comment]
agreed

**Comment by ekralc** [comment]
This has been addressed before in #1372

It seems that this is an edge case where 'multiline' strings that are actually a single line aren't caught.

For example, this fails as expected with a validation error:
```toml
description = """\
Multiple
line
description
"""
```
